solver/Cargo.toml

```diff
@@ -10,4 +10,4 @@ maplit = "1.0"
 model = { path = "../model" }
 primitive-types = "0.7"
 reqwest = { version = "0.10", features = ["json"] }
-tokio = { version = "0.2", features =[ "macros"] }
+tokio = { version = "0.2", features =[ "macros", "time"] }
```

Comment:
I think I saw this dependency introduced in another recent PR. You might find conflicts with this at some point... but no big deal